internal/db/migrations.go

```diff
@@ -317,4 +317,29 @@ var sqlMigrations = map[string]string{
 		ALTER TABLE project_commitments
 			ADD COLUMN renew_context_json JSONB;
 	`,
+	"052_project_resources_forbidden.down.sql": `
+		ALTER TABLE project_resources
+			ADD COLUMN min_quota_from_backend BIGINT DEFAULT NULL,
+			ADD COLUMN max_quota_from_backend BIGINT DEFAULT NULL;
+			
+		UPDATE project_resources
+			SET max_quota_from_backend = 0
+			WHERE forbidden = TRUE;
+		
+		ALTER TABLE project_resources
+			DROP COLUMN forbidden;
+	`,
+	"052_project_resources_forbidden.up.sql": `
+		ALTER TABLE project_resources
+			ADD COLUMN forbidden BOOLEAN DEFAULT FALSE;
```

Comment:
For the same reason:
```suggestion
			ADD COLUMN forbidden BOOLEAN NOT NULL DEFAULT FALSE;
```